docs/docs/learn/introduction/op-stack.md

```diff
@@ -24,7 +25,7 @@ BOB is adopting several changes compared to the standard OP Stack:
 
 BOB will have its own governance to reach community decisions and enact changes.
 
-If the BOB Rollup chooses to be comptabile with the Superchain, then parts of BOB will be subject to OP governance with the Superchain upgrade. An OP Security Council will be capable of enacting security upgrades of the OP chain and L1 and L2 bridge contracts.
```

Comment:
@nud3l Hey just wondering why you closed this PR, seems this typo hasn't been fixed yet.